### PR TITLE
Update Azure EventHubs PubSub to use AAD for checkpoint storage

### PR DESCRIPTION
**Pull request by 767829413**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: pubsub/mqtt/mqtt.go [hunk 1]
```diff
@@ -216,7 +216,7 @@ func (m *mqttPubSub) Subscribe(req pubsub.SubscribeRequest, handler pubsub.Handl
 		token := m.consumer.SubscribeMultiple(
 			m.topics,
 			func(client mqtt.Client, mqttMsg mqtt.Message) {
-				mqttMsg.AutoAckOff()
+				// mqttMsg.AutoAckOff()
```
Comment (surenderssm) [inline pubsub/mqtt/mqtt.go:219]: Is this intentional?

**Comment by berndverst** [comment]
This is a duplicate of my PR opened 9 hours before yours.

https://github.com/dapr/components-contrib/pull/1578

**Comment by berndverst** [comment]
If you want additional changes beyond what is in #1578 please open a PR after that is merged.

**Comment by berndverst** [comment]
Also #1572

File: pubsub/mqtt/mqtt.go (hunk 1 above)
Comment (767829413) [reply pubsub/mqtt/mqtt.go:219]: Hello, this should not be intentional, the source code should be restored

**Comment by 767829413** [comment]
thanks, i understand

**Pull request closed without merging**